update to community pool

evanorti opened the pull request:
Update to mentions of the community pool in the docs. This update outlines that the community pool is no longer funded, as well as other C-5 changes.

evanorti committed:
Update dev/Readme for C-5 changes
Grammar and spelling edits.

evanorti committed:
Update distribution pages to reflect C-5 changes.
terrad/distribution and dev/spec-distribution were edited for community 
pool changes, spelling, and grammar.

evanorti committed:
Update governance pages to reflect C-5
dev/spec-governance, docs/governance, terrad/governance were reviewed to 
reflect C-5 changes, as well as spelling and grammar.

evanorti committed:
edit sentence flow

stevenTerra commented on docs/dev/README.md:
```diff
@@ -96,19 +96,19 @@ The following processes get executed during each block transition:
 There are two types of tokens that can be held by accounts and wallets in the Terra protocol:
 
-1. **Terra Stablecoins** are transactional assets that are track the exchange rate of various fiat currencies. By convention, given a fiat currency, the Terra peg that corresponds to it is Terra-`<3-letter ISO4217 currency-code>` (see [here](https://www.xe.com/iso4217.php)) abbreviated `<country-code>T`, where the `T` replaces the currency's designator. For instance, TerraKRW is the peg for the K ...
+1. **Terra Stablecoins** are transactional assets that track the exchange rate of various fiat currencies. By convention, given a fiat currency, the Terra peg that corresponds to it is Terra-`<3-letter ISO4217 currency-code>` (see [here](https://www.xe.com/iso4217.php)) abbreviated `<country-code>T`, where the `T` replaces the currency's designator. For instance, TerraKRW is the peg for the Korea ...
```
I found this section difficult to parse. I've tried to improve on it, but I'm not sure whether I've made it any better. 

**Terra stablecoins** are transactional assets that track the exchange rate of various fiat currencies. By convention, given a fiat currency, the Terra peg that corresponds to it is Terra<currency-code>, where <currency-code> is the [ISO 4217 code](https://www.xe.com/iso4217.php). Its abbreviation is <country-code>T, where <country-code> is the first two characters of the <currency-code>, and T replaces the currency's designator. For example, TerraKRW is the peg for the Korean won, and its abbreviation is KRT.

evanorti replied:
Re-worded.

stevenTerra commented on docs/dev/README.md:
```diff
@@ -96,19 +96,19 @@ The following processes get executed during each block transition:
    The flagship, standard Terra currency is TerraSDR, or SDT, the peg to the IMF's Special Drawing Rights. The protocol will use SDT as its default, "base" currency to do calculations and setting standards.
 
-2. **Luna**, the native staking asset that entitles the staking delegator to mining rewards (including exchange rate ballot rewards) if bonded to an active validator. Luna is also is necessary for making governance proposals and collateralizing the Terra economy.
+2. **Luna**, the native staking asset that entitles the staking delegator to mining rewards (including exchange rate ballot rewards) if bonded to an active validator. Luna is also necessary for making governance proposals and collateralizing the Terra economy.
```
replace the parentheses with commas

stevenTerra commented on docs/dev/spec-distribution.md:
```diff
@@ -14,12 +14,16 @@ The `Distribution` module describes a mechanism that keeps track of collected fe
 :::
 
-Collected rewards are pooled globally and divided out passively to validators and delegators. Each validator has the opportunity to charge commission to the delegators on the rewards collected on behalf of the delegators. Fees are collected directly into a global reward pool and validator proposer-reward pool. Due to the nature of passive accounting, whenever changes to parameters which affect th ...
+Collected rewards are pooled globally and divided out passively to validators and delegators. Each validator has the opportunity to charge delegators commission on the rewards collected on behalf of the delegators. Fees are collected directly into a global reward pool and a validator proposer-reward pool. Due to the nature of passive accounting, whenever changes to parameters which affect the rat ...
```
Remove "out"

stevenTerra commented on docs/dev/spec-distribution.md:
```diff
@@ -177,9 +181,9 @@ type CommunityPoolSpendProposal struct {
 At the beginning of the block, the Distribution module will set the proposer for determining distribution during endblock and distribute rewards for the previous block.
 
-The fees received are transferred to the Distribution `ModuleAccount`, as it's the account the one who keeps track of the flow of coins in (as in this case) and out the module. The fees are also allocated to the proposer, community fund and global pool. When the validator is the proposer of the round, that validator (and their delegators) receives between 1% and 5% of fee rewards, the reserve [Co ...
+The fees received are transferred to the Distribution `ModuleAccount`, as which keeps track of the flow of coins in (as in this case) and out the module. The fees are also allocated to the proposer, community fund and global pool. When the validator is the proposer of the round, that validator (and their delegators) receives between 1% and 5% of fee rewards, the reserve [Community Tax](#community ...
```
There are a few changes to be made in this paragraph: The fees received are transferred to the Distribution `ModuleAccount`, which tracks the flow of coins in, as in this case, and out the module. The fees are also allocated to the proposer, community fund, and global pool. When the validator is the proposer of the round, that validator and its delegators receive 1-5% of the fee rewards, the reserve [Community Tax](#communitytax) is charged, and then the remainder is distributed proportionally by voting power to all bonded validators independent of whether they voted (social distribution). Social distribution is applied to the proposer validator in addition to the proposer reward.


I'm not sure what to do with the parenthetical phrase. I want to remove the parentheses, but I don't know enough about social distribution to determine where to place it.

evanorti replied:
Changed.

stevenTerra commented on docs/dev/spec-distribution.md:
```diff
@@ -177,9 +181,9 @@ type CommunityPoolSpendProposal struct {
+The fees received are transferred to the Distribution `ModuleAccount`, as which keeps track of the flow of coins in (as in this case) and out the module. The fees are also allocated to the proposer, community fund and global pool. When the validator is the proposer of the round, that validator (and their delegators) receives between 1% and 5% of fee rewards, the reserve [Community Tax](#community ...
 
-The amount of proposer reward is calculated from pre-commits Tendermint messages in order to incentivize validators to wait and include additional pre-commits in the block. All provision rewards are added to a provision reward pool which validator holds individually (`ValidatorDistribution.ProvisionsRewardPool`).
+The amount of proposer reward is calculated from pre-commits Tendermint messages in order to incentivize validators to wait and include additional pre-commits in the block. All provision rewards are added to a provision reward pool which each validator holds individually (`ValidatorDistribution.ProvisionsRewardPool`).
```
- No need to hyphenate words that begin with pre unless doing so would cause confusion
- The phrase "in order to" can be shortened to just "to"
- Add a comma after pool in the last sentence

stevenTerra commented on docs/dev/spec-governance.md:
```diff
@@ -14,9 +14,9 @@ The following is the governance proposal procedure:
+After a proposal is submitted, it enters the deposit period, where it must reach a total minimum deposit of 512 Luna within 2 weeks from the time of its submission. The deposit threshold is reached when the sum of the initial deposit (from the proposer) and the deposits from all other interested network participants meets or exceeds 512 Luna.
 
-The deposit is only required as a form of spam protection, and the network will refund Luna deposits for proposals that have passed or have rejected but not vetoed.
+Deposits are only required as a form of spam protection, and the network will refund Luna deposits for proposals that pass or are rejected. Deposits are not refunded to proposals that are rejected with veto.
```
We can make this one more active, such as, "Deposits are required only to protect against spam. The network refunds deposits for proposals that pass or fail except when a proposal is vetoed. If a proposal is vetoed, deposits are not refunded."

stevenTerra commented on docs/dev/spec-governance.md:
```diff
@@ -31,25 +31,29 @@ Voting is done by holders of bonded LUNA on a 1 bonded LUNA = 1 vote basis. As s
 ### Tallying
 
-Three conditions must be met for a proposal to pass.
+Three conditions must be met for a proposal to pass:
```
A good rule: Avoid stating quantities of conditions. Leave them open-ended so that you can add to them or subtract from them without needing to update the quantity every time you update the list. So, in this case, you might say, "For a proposal to pass, the following conditions must be met:"

stevenTerra commented on docs/dev/spec-governance.md:
```diff
@@ -31,25 +31,29 @@ Voting is done by holders of bonded LUNA on a 1 bonded LUNA = 1 vote basis. As s
 $$\frac{Yes}{Yes + No + NoWithVeto} \gt T$$
 
-If any of the above conditions are not met, the proposal is rejected. The deposit associated with rejected proposals are not refunded and will be transferred instead to the Community Pool. The parameters `quorum`, `veto`, and `threshold` exist as blockchain parameters within the Governance module.
+If any of the above conditions are not met, the proposal is rejected. Proposals that get rejected with veto do not get their deposits refunded. The parameters `quorum`, `veto`, and `threshold` exist as blockchain parameters within the Governance module.
```
For better accessibility, say, for people who use screen readers, try to avoid spatial references like "above" and "below." In this particular case, you could replace above with previous.

stevenTerra commented on docs/dev/spec-governance.md:
```diff
@@ -73,43 +77,9 @@ type Proposal struct {
+A `Proposal` is a data structure representing a petition for a change that is submitted to the blockchain alongside a deposit. Once its deposit reaches a certain value ([`MinDeposit`](#mindeposit)), the proposal is confirmed and voting opens. Bonded Luna holders can then send [`TxGovVote`]() transactions to vote on the proposal. Terra currently follows a simple voting scheme of 1 Bonded Luna = 1  ...
 
-The `Content` on a proposal is an interface which contains the information about the `Proposal` such as the `title`, `description`, and any notable changes. Also, this `Content` type can by implemented by any module. The `Content`'s `ProposalRoute` returns a string which must be used to route the `Content`'s Handler in the Governance keeper. This allows the governance keeper to execute proposal l ...
+The `Content` on a proposal is the interface that contains the information about the `Proposal` such as the `title`, `description`, and any notable changes. A `Content` type can be implemented by any module. The `ProposalRoute` of the `Content` returns a string which must be used to route the handler of the `Content` in the Governance keeper. This allows the governance keeper to execute proposal  ...
```
- add a comma after between `Proposal` and such in the first sentence
- Replace the naked this

stevenTerra commented on docs/dev/spec-governance.md:
````diff
@@ -215,7 +124,7 @@ type TextProposal struct {
 ```
 
-Text Proposals are used for creating general-purpose petitions, such as asking the Core team to implement a specific feature. The community can reference a passed Text Proposal to the core developers to indicate that a feature (requiring potentially a soft or hard fork) is in significant demand.
+Text Proposals are used to create general-purpose petitions, such as asking the Core team to implement a specific feature. The community can reference a passed Text Proposal to the core developers to indicate that a feature (requiring potentially a soft or hard fork) is in significant demand.
````
- A mismatch of case in the term "core." One instance is uppercase, second instance is lowercase. I tend toward putting almost all of these types of terms in lowercase.
- Remove the parentheses. In this instance, you probably have to say "a feature that potentially requires a soft or hard fork"

stevenTerra commented on docs/dev/spec-governance.md:
```diff
@@ -234,16 +143,16 @@ type ParamChange struct {
 
 ::: danger
-Software Upgrade Proposals also exist due to inheritance from Cosmos SDK but are for the moment considered unavailable, as they have not yet been implemented. They thus share the same semantics as a simple Text Proposal. It is strongly advised to not submit these types of proposals at the risk of losing your Luna deposit.
+Software Upgrade Proposals exist due to inheritance from Cosmos SDK but are for the moment considered unavailable, as they have not yet been implemented. They thus share the same semantics as a simple Text Proposal. It is strongly advised to not submit these types of proposals at the risk of losing your Luna deposit.
```
Here's another instance of case. I don't think something like software upgrade proposals needs to be capitalized. We can reconstruct this short paragraph in a couple of useful ways, such as, "Software upgrade proposals exist because they are inherited from the Cosmos SDK, but they are temporarily unavailable because they have not been implemented yet. Therefore, they share the same semantics as a simple text proposal. If you submit this type of proposal, you might lose your Luna deposit.

stevenTerra commented on docs/governance.md:
```diff
@@ -4,15 +4,15 @@ Governance is the process by which Terra network participants can effect change
-- `TaxRateUpdateProposal`: update Tax Rate monetary policy lever
-- `RewardWeightUpdateProposal`: update Reward Weight monetary policy lever
-- `CommunityPoolSpendProposal`: disbursement from the Distribution module's Community Pool
+- `ParameterChangeProposal`: a change to blockchain parameters (defined in each module).
```
- Remove the parentheses. Here you could just say, "a change to blockchain parameters, which are defined in each module
- Some of the items in the list end in periods, and some don't. Generally, add a period only if the statement forms a complete sentence, which none of these do. Remove the period from each

stevenTerra commented on docs/terrad/governance.md:
```diff
@@ -4,13 +4,13 @@
 ### Proposals
 
-Once created, you can now query information of the proposal:
+Once created, the information of a single proposal can be queried:
```
After a proposal is created, you can query it by running the following command:

stevenTerra commented on docs/terrad/governance.md:
```diff
@@ -144,14 +144,10 @@ Where `proposal.json` is a file with the following schema:
-that any `value` change is valid (ie. correct type and within bounds) for its
-respective parameter, eg. `MaxValidators` should be an integer and not a decimal.
+::: warning Note:
+Parameter changes are _evaluated_ but not _validated_. Proposers must ensure that any `value` change to a parameter is of the correct type and within the correct bounds. For example, `MaxValidators` is an integer and not a decimal.
```
The italics seem unnecessary.

evanorti committed:
Update dev/readme to reflect changes.
Grammar changes, clarity.

evanorti committed:
Beginblock distribution updated
Changed wording to begin block section for clarity. Community pool 
updated to reflect that it is no longer funded.

evanorti committed:
Update grammar and verbiage for clarity

evanorti committed:
Governance update

evanorti committed:
Merge remote-tracking branch 'upstream/columbus-5-demo' into Community-pool-update
